Anlab.Jobs.MoneyMovement/Program.cs

```diff
@@ -61,6 +61,12 @@ static void Main(string[] args)
             }
             Console.WriteLine("Done Credit Cards");
 
+            result = Task.Run(() => SlothService.MoneyMovedCheck(financialSettings)).Result;

```

Comment:
@jSylvestre sorta just FYI but if you want you can run these two in parallel since they are orthogonal, using Task.WhenAll().